huaweicloud/resource_huaweicloud_nat_dnat_rule_v2.go

```diff
@@ -87,6 +87,13 @@ func ResourceNatDnatRuleV2() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
```

Comment:
should we add a validatefunc here as there are some limitations in the doc

Reply:
the limitation is too simple, so we can ignore the validatation in code.